src/content/docs/licenses/license-information/faq/new-relic-one-pricing-plan-frequently-asked-questions.mdx

```diff
@@ -66,7 +66,7 @@ These FAQs are meant to serve as a supplement to the main [New Relic pricing doc
     id="FAQ_6"
     title="Q: Can Event extended retention, additional Synthetics Checks, or additional New Relic Edge data be added to my account?"
   >
-    **A:** For additional retention for Events or Logs beyond the Standard Data Retention, contact your New Relic account executive. Note: Minimum requirement for Extended Retention requires an Annual Pool of Funds subscription.
+    **A:** For additional retention for Events or Logs beyond the Standard Data Retention, contact your New Relic account executive. Note: Minimum requirement for Extended Retention requires a [contract](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#usage-plans).
```

Comment:
It feels weird to have a single word link like this.

maybe something more like: "requires a [Volume, Savings, or other contract]"